internal/lsp/server.go

```diff
@@ -936,33 +932,24 @@ func (l *LanguageServer) processTextContentUpdate(
 	ctx context.Context,
```

Comment:
See note above, this function now just attempts to update the parse if the content changed.